app/controllers/users_controller.rb

```diff
@@ -1,18 +1,53 @@
 class UsersController < ApplicationController
-  before_action :set_user
+  before_action :set_user, except: [:index, :new, :create]
+  after_action :verify_authorized, except: :index
+  after_action :verify_policy_scoped, only: :index
+
+  def index
+    @users = policy_scope(User)
+  end
+
+  def show
+    authorize @user
+  end
+
+  def new
+    @user = User.new
+    authorize @user
+  end
+
+  def create
+    @user = User.new user_params
+
+    authorize @user
+
+    if @user.save
+      current_user.organisations.each do |organisation|
```

Comment:
Here `current_user` is `@user`, right? Maybe using the same variable name will be cleaner?